Describe Date Range Parameters (#31)
serge-andreyev/dotnet-eud: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/use-report-parameters.md

---
# Use Report Parameters

The topics in this section describe how to use parameters in a report:

* [Parameters Overview](use-report-parameters/parameters-overview.md)

	Describes how to create and use report parameters.

* [Request and Pass Report Parameter Values](use-report-parameters/request-and-pass-report-parameter-values.md)

> 	Illustrates how to assign the default and custom values to a report's parameters and describes the editors that are used to request these values in a Print Preview.

Reviewer: ```suggestion
	Illustrates how to assign default and custom values to a report's parameters and describes the editors that are used to request these values in a Print Preview.
```

* [Multi-Value and Cascading Report Parameters](use-report-parameters/multi-value-and-cascading-parameters.md)

> 	Explains how to configure report parameters so that they accept multiple values, and how to filter a parameter's values based on another parameter's value.

Reviewer: ```suggestion
	Explains how to configure report parameters to accept multiple values and filter a parameter's values based on another parameter's value.
```

* [Date Range Parameters](use-report-parameters/date-range-parameters.md)

	Describes how to create date range parameters to filter report data by a specific time period.

* [Query Parameters](use-report-parameters/query-parameters.md)

> 	Details how to link report parameters with query parameters defined in your report's data source.

Reviewer: ```suggestion
	Explains how to link report parameters to query parameters defined in your report's data source.
```

Author: Ok, changed
